React/Base/RCTConvert.m

```diff
@@ -882,7 +882,7 @@ + (UIImage *)UIImage:(id)json
   NSString *scheme = URL.scheme.lowercaseString;
   if ([scheme isEqualToString:@"file"]) {
     NSString *assetName = RCTBundlePathForURL(URL);
-    image = [UIImage imageNamed:assetName];
+    image = [UIImage imageNamed:[NSString stringWithFormat:@"%@", assetName]];
```

Comment:
I agree with @ide. I suggest changing this to:

    image = assetName ? [UIImage imageNamed:assetName] : nil;